templates/CRM/Member/Form/UimodsMembershipView.tpl

```diff
@@ -0,0 +1,102 @@
+{*
+ +--------------------------------------------------------------------+
+ | CiviCRM version 4.6                                                |
```

Comment:
We currently run on Civi 5.7, so please use the template from 5.7.3 as the starting point for this.